java/token-requestor-client/pom.xml

```diff
@@ -33,6 +33,7 @@
                         </goals>
                         <configuration>
                             <configOptions>
+                                <useJakartaEe>true</useJakartaEe>
```

Comment:
So did this have effect or not?

Reply:
Seems like it did not, however, this is something that is fixed in unreleased versions, so I don't see why we shouldn't have it (for when the time comes...)